src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -572,18 +572,25 @@ protected function getContainerBaseClass()
      */
     protected function initializeContainer()
     {
+        $asFiles = true;
         $class = $this->getContainerClass();
         $cacheDir = $this->warmupDir ?: $this->getCacheDir();
         $cache = new ConfigCache($cacheDir.'/'.$class.'.php', $this->debug);
         $oldContainer = null;
         if ($fresh = $cache->isFresh()) {
+            if (!$asFiles) {
+                require_once $cache->getPath();
+                $this->container = new $class();
+                $this->container->set('kernel', $this);
+
+                return;
+            }
             // Silence E_WARNING to ignore "include" failures - don't use "@" to prevent silencing fatal errors
             $errorLevel = error_reporting(\E_ALL ^ \E_WARNING);
             $fresh = $oldContainer = false;
             try {
                 if (file_exists($cache->getPath()) && \is_object($this->container = include $cache->getPath())) {
                     $this->container->set('kernel', $this);
-                    $oldContainer = $this->container;
```

Comment:
This var was useless, because at the new line we have `$fresh=true` and later `if ($fresh} { return; }`

Reply:
unless the line above throws an exception, which is why this code is how it is

Reply:
:+1:

Reply:
I don't understand: 
* If the line above throw an exception, then we can not execute this line.
* If the line above does not throw exception, so we execute this line. But this line can not fail. So we execute the next line: `$fresh=true` and later `if ($fresh} { return; }`. So to me this is dead code.